src/Umbraco.Core/ObjectExtensions.cs

```diff
@@ -652,5 +644,69 @@ internal static Guid AsGuid(this object value)
 	    {
 	        return value is Guid ? (Guid) value : Guid.Empty;
 	    }
-	}
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static string NormalizeNumberDecimalSeparator(string s)
+        {
+            var normalized = System.Threading.Thread.CurrentThread.CurrentCulture.NumberFormat.NumberDecimalSeparator[0];
+            return s.ReplaceMany(NumberDecimalSeparatorsToNormalize, normalized);
+        }
+
+        // gets a converter for source, that can convert to target, or null if none exists

```

Comment:
If adding comments why not proper intellisense ones so you can see them from the calling code?

Reply:
Weird tradition of mine for entirely private methods. But I think you're right on that one.